python/cudf/cudf/core/tools/datetimes.py

```diff
@@ -331,3 +332,71 @@ def get_units(value):
         return _unit_map[value.lower()]
 
     return value
+
+
+class Container(object):
+    def __init__(self, arg):
+        self.arg = arg
```

Comment:
Why do we need this class?

Reply:
We don't. Leftover code from a failed experiment. Not sure how I didn't notice that.

Reply:
Backpedaling the previous comment I ended up actually using something like this in conjunction with overriding `__setattr__`.